.github/actions/setup-deps/action.yaml

```diff
@@ -133,6 +135,7 @@ runs:
           ${{ inputs.scikit-learn }}
           ${{ inputs.seaborn }}
           ${{ inputs.tidynamics }}
+          ${{ inputs.dask }}
```

Comment:
Can you move this to above the distopia entry? These are meant to be alphabetically ordered.